### PR TITLE
Fix sidebar icons alignment

### PR DESCRIPTION
**Pull request by Usiel-03A7**
## Description

Fix sidebar icons alingment.

## Evidence

Before
image
After
image

## Link Issue (if applicable)
  -

### CODE REVIEW

File: src/components/sidebar/OptionsPanel/OptionItem/index.jsx [hunk 1]
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
 import styles from "./OptionItem.module.css";
 
 export const OptionItem = ({ optionText, path, children }) => {
-  const [isActive, setActive] = useState(false)
+  const [isActive, setActive] = useState(false);
```
Comment (AlanAnguiano) [inline src/components/sidebar/OptionsPanel/OptionItem/index.jsx]: Avoid using ;

File: src/components/sidebar/OptionsPanel/OptionItem/index.jsx [hunk 2]
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
...
 
   const getActiveStyle = () => {
-    return isActive ? styles.optionPanel__itemActive : '' 
-  }
-  
-  useEffect(()=>{
-    const currentPath = window.location.pathname
+    return isActive ? styles.optionPanel__itemActive : "";
```
Comment (AlanAnguiano) [inline src/components/sidebar/OptionsPanel/OptionItem/index.jsx]: Same as above

File: src/components/sidebar/OptionsPanel/OptionItem/index.jsx [hunk 3]
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
...
+  };
 
-    if(currentPath === path) {
-      setActive(true)
+  useEffect(() => {
+    const currentPath = window.location.pathname;
+
+    if (currentPath === path) {
+      setActive(true);
```
Comment (AlanAnguiano) [inline src/components/sidebar/OptionsPanel/OptionItem/index.jsx]: Same

File: src/components/sidebar/OptionsPanel/OptionItem/index.jsx [hunk 4]
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
...
+      <div className={`${styles.optionPanel__item} ${getActiveStyle()}`}>
+        <i>{children}</i>
+        <p>{optionText}</p>
       </div>
     </>
-  )
-}
\ No newline at end of file
+  );
+};
```
Comment (AlanAnguiano) [inline src/components/sidebar/OptionsPanel/OptionItem/index.jsx]: Same, and also in all the lines that you find

**Review by AlanAnguiano — DISMISSED** [review]

File: src/components/sidebar/OptionsPanel/OptionItem/index.jsx (hunk 1 above)
Comment (Usiel-03A7) [reply src/components/sidebar/OptionsPanel/OptionItem/index.jsx]: done.

**Review by AlejandroCuriel — APPROVED** [review]
LGTM BRO! 💯

**Review by AlanAnguiano — APPROVED** [review]
Shipt it :shipit:

**Pull request merged**